### ISSUE TITLE
typo in lsgan/model.py

### ISSUE THREAD

**Issue by Emilio856** [thisisiron/dcgan-tf#4]
Lines 48 and 52 build the third batch normalization layer into the model. In line 52 it should be the fourth one, so it shoul be **x = self.batch_norm_4(x, training)** instead of **x = self.batch_norm_3(x, training)**
